From d41cb9651bf22acebaae5b36e7bad7a1e939dc60 Mon Sep 17 00:00:00 2001
From: Sandor Yu <R01008@freescale.com>
Date: Thu, 3 Jul 2014 17:47:34 +0800
Subject: [PATCH 0443/1074] ENGR00321114 dts: Recpvery gpr item in vadc dts
 item

Gpr item is remove from vadc dts by dcic patch incorrectly.
It should add back, otherwise vadc driver can not initialized.

Signed-off-by: Sandor Yu <R01008@freescale.com>
[Original patch taken from git://git.freescale.com/imx/linux-2.6-imx.git]
Signed-off-by: Biyao Zhai <biyao.zhai@windriver.com>
---
 arch/arm/boot/dts/imx6sx.dtsi |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/arch/arm/boot/dts/imx6sx.dtsi b/arch/arm/boot/dts/imx6sx.dtsi
index 1f49338..e0ad7a0 100644
--- a/arch/arm/boot/dts/imx6sx.dtsi
+++ b/arch/arm/boot/dts/imx6sx.dtsi
@@ -906,6 +906,7 @@
 					clocks = <&clks IMX6SX_CLK_VADC>,
 							<&clks IMX6SX_CLK_CSI>;
 					clock-names = "vadc", "csi";
+					gpr = <&gpr>;
 					csi_id = <0>;
 					status = "disabled";
 				};
-- 
1.7.5.4

